lib/src/doc/document.rs

```diff
@@ -83,7 +83,7 @@ impl<'a> Document<'a> {
 		&self,
 		opt: &Options,
 		txn: &Transaction,
-	) -> Result<Arc<Vec<DefineTableStatement>>, Error> {
+	) -> Result<Arc<[DefineTableStatement]>, Error> {
```

Comment:
Unless you plan to extract the `Vec` out of the `Arc` if the reference count drops to 1, `Arc<[T]>` has a better, less-indirected memory layout than `Arc<Vec<T>>`, while preserving functionally the same API.